src/ecrc-frontend/public/404.html

```diff
@@ -51,6 +51,8 @@
     <title>Criminal Record Check</title>
   </head>
   <body>
-    <div>This is a 404.</div>
+    <h1>404</h1>
+    <p>There is currently an issue with the server.</p>
```

Comment:
isnt 404 the page/resource you are requesting doesnt exist... not server issue (500)

Reply:
The HTTP 404, 404 Not Found, 404, Page Not Found, or Server Not Found error message is a Hypertext Transfer Protocol standard response code

Reply:
ok gotcha, thanks for clarity